src/rebar3_elixir_compile_resource.erl

```diff
@@ -79,10 +79,14 @@ fetch({elixir, Name_, Vsn_}, CDN) ->
     Vsn  = rebar3_elixir_compile_util:to_binary(Vsn_),
     case filelib:is_dir(Dir) of
         false ->
-            Package = binary_to_list(<<Name/binary, "-", Vsn/binary, ".tar">>),
-            Url = string:join([CDN, Package], "/"),
-            case request(Url) of
-                {ok, Binary} ->
+            Config = #{
+              http_adapter => hex_http_httpc,
+              http_adapter_config => #{profile => default},
+              http_user_agent_fragment => <<"(httpc)">>,
```

Comment:
Consider adding your library + version into user agent string, might be helpful later if we need to parse logs.